### PR TITLE
[ENH] Migrate l10n_nl_partner_name to 8.0

### PR DESCRIPTION
**Pull request by NL66278**

### CODE REVIEW

**Comment by hbrunn** [comment]
please add a file `oca_dependencies.txt` as described in https://github.com/OCA/maintainer-quality-tools/blob/master/travis/clone_oca_dependencies to have travis pull partner-contact as dependency

**Comment by NL66278** [comment]
@hbrunn Holger is there any example of such a dependencies files anywhere?

**Comment by hbrunn** [comment]
https://github.com/OCA/connector-interfaces/pull/9/files - simply one line containing the project name

File: l10n_nl_partner_name/data/migration.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hbrunn) [inline l10n_nl_partner_name/data/migration.xml]: you can remove this file, as there won't be a 8.0 database using the old name

File: l10n_nl_partner_name/model/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            context.get(
+                'name_format',
+                "${p.firstname or p.initials or ''}"
+                "${(p.firstname or p.initials) and ' ' or ''}"
+                "${p.infix or ''}${p.infix and ' ' or ''}${p.lastname}"))
+        name = name_template.render(p=partner)
+        return (name[0] + name[1:]) if name else ''
+
+    def _migrate_partner_name_dutch_name_change(
+            self, cr, uid, old_name, context=None):
```
Comment (hbrunn) [inline l10n_nl_partner_name/model/res_partner.py]: this can also go away with migration.xml

**Comment by hbrunn** [comment]
:+1: thanks!

**Comment by NL66278** [comment]
Closed in favor of #16

**Pull request closed without merging**